monkey/infection_monkey/network_scanning/ssh_fingerprinter.py

```diff
@@ -38,6 +48,8 @@ def get_host_fingerprint(
                     )
                 )
 
+        self._publish_fingerprinting_event(host, timestamp, services)
```

Comment:
I'm not sure that this should publish an event. It's not doing anything active. In other words, there's nothing observable from outside Infection Monkey (i.e. no "event" occurs).

Reply:
Although, I suppose it has to, otherwise the Island won't know about he service. Seems like a design issue, but we can address it if/when we ever refactor scanning.

Reply:
Good point, removing it